packages/webapp/src/components/Task/PureTaskTypeSelection/PureTaskTypeSelection.jsx

```diff
@@ -77,6 +79,9 @@ export const PureTaskTypeSelection = ({
   register(TASK_TYPE_ID);
   const selected_task_type = watch(TASK_TYPE_ID);
 
+  const isMakingCropTask = location?.state?.management_plan_id ?? false;
```

Comment:
Did you use `??` to retain `0`? Later, since you're doing `isMakingCropTask && ...`, 0 would act as `false` anyway.
I think it could simply be:
```
const isMakingCropTask = !!location?.state?.management_plan_id;
```

Reply:
🙏  thanks that is better!